src/test/java/edu/hawaii/its/groupings/configuration/OotbStaticUserAuthenticationFilterTest.java

```diff
@@ -0,0 +1,60 @@
+package edu.hawaii.its.groupings.configuration;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.mockito.Mockito.*;
```

Comment:
Make sure to remove * imports from all files. Also it seems that your code style is not enabled because static imports are usually on the top. Thanks!

Reply:
Replace the all import here from Mockito with the imports for when(), verify(), thenReturn(), never(), and any().

Reply:
Oh I don't know why it was not updated by git push, but I fixed it